codegen-client-test/model/rest-xml-extras.smithy

```diff
@@ -10,6 +10,8 @@ use smithy.test#httpRequestTests
 /// A REST XML service that sends XML requests and responses.
 @service(sdkId: "Rest XML Protocol")
 @restXml
+@httpApiKeyAuth(scheme: "Apikey", name: "Authorization", in: "header")
+@auth([httpApiKeyAuth])
```

Comment:
I think the Kotlin unit test should be sufficient to test this, so I don't think this is necessary.

Reply:
removed